user-guide/Getting_started/DaaS/Creating_a_DMS_on_dataminer_services.md

```diff
@@ -32,6 +32,10 @@ uid: Creating_a_DMS_on_dataminer_services
    > [!NOTE]
    > Only lowercase alphanumerical characters (a-z and 0-9) are allowed in the URL, and it cannot consist of numbers only.
 
+1. Optionally, in the *Time zone* box, select the time zone for your DataMiner System.
```

Comment:
The label of the time zone will be change in a upcomming release.
We changed it to 'DataMiner System time zone' so that it is inline with the other input fields about the DMS you are deploying.

```suggestion
1. Optionally, in the *DataMiner System time zone* box, select the time zone for your DataMiner System.
```